src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
```

Comment:
> Write an interface Figure and 3 classes implementing it so that every figure has:
> - a shape (triangle, circle or rectangle);
> - a color (red, green or blue);

```suggestion
  shape: Shape= 'triangle';

  constructor(
    public color: Color,
```
You need to create Color and Shape **type** and give tham key values (keywords)🤔